pytorch_lightning/utilities/argparse.py

```diff
@@ -135,29 +136,49 @@ def get_init_arguments_and_types(cls) -> List[Tuple[str, Tuple, Any]]:
     return name_type_default
 
 
-def add_argparse_args(cls, parent_parser: ArgumentParser) -> ArgumentParser:
+def add_argparse_args(
+    cls,
+    parent_parser: Union[ArgumentParser, ArgumentGroup],
+    *,
+    inplace=False,
+) -> Union[ArgumentParser, ArgumentGroup]:
```

Comment:
Update the signatures of the functions that call this one, please.

Reply:
Done. (No longer relevant, I think)